src/workspacer.TitleBar/TitleBarPlugin.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace workspacer.TitleBar
+{
+    public class TitleBarPlugin : IPlugin
+    {
+        private TitleBarPluginConfig _config;
+        private IConfigContext _context;
+
+        public TitleBarPlugin()
+        {
+            _config = new TitleBarPluginConfig();
+        }
+
+        public TitleBarPlugin(TitleBarPluginConfig config)
+        {
+            _config = config;
+        }
+
+        public void AfterConfig(IConfigContext context)
+        {
+            _context = context;
+            _context.Workspaces.WindowAdded += WindowAdded;
+        }
+
+        private void SetStyle(IWindow window, TitleBarStyle settings)
+        {
+            var style = (Win32.WS)Win32.GetWindowLongPtr(window.Handle, Win32.GWL_STYLE);
+            if (settings.ShowTitleBar)
+            {
+                style |= Win32.WS.WS_CAPTION;
+            }
+            else
+            {
+                style &= ~Win32.WS.WS_CAPTION;
+            }
+
+            if (settings.ShowSizingBorder)
+            {
+                style |= Win32.WS.WS_THICKFRAME;
+            }
+            else
+            {
+                style &= ~Win32.WS.WS_THICKFRAME;
+            }
+            Win32.SetWindowLongPtr(window.Handle, Win32.GWL_STYLE, (IntPtr)style);
```

Comment:
If manipulation of the `style`-variable was extracted to its own method, this code could quite easily be tested through unit-tests.

Reply:
I added unit tests for updating the `Win32.WS style` in 5d13fc11c2849e44e925cde08361aebb3636248c.

That commit also extracts extracts the manipulation of the `style` variable to its own static method.

Reply:
LGTM!